### PR TITLE
Add small changes to selenium tests to according to client

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Add small fixes to selenium tests for success passing:
  * AutocompleteFeaturesInEditorTest
  * FormatterTest
  * CheckIntelligenceCommandFromToolbarTest
  * CommandsEditorTest

### CODE REVIEW

**Review by mmorhun — APPROVED** [review]

**Comment by musienko-maxim** [comment]
The title of commit is not clear, please, follow our convention for this one. More details about it you can know @garagatyi

**Comment by artaleks9** [comment]
ci-build

**Review by garagatyi — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void checkButtonsOnToolbar() {
     checkTestAppAndReturnToIde(currentWindow, "Enter your name:");
     Assert.assertTrue(commandsToolbar.getTimerValue().matches("\\d\\d:\\d\\d"));
     Assert.assertTrue(commandsToolbar.getNumOfProcessCounter().equals("#2"));
-    commandsToolbar.clickOnPreviewCommandBtnAndSelectUrl("ws-machine:8080");
+    commandsToolbar.clickOnPreviewCommandBtnAndSelectUrl("dev-machine:8080");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java:88]: @garagatyi: question - should hardcoded port 8080 be maintainable here for the case when Che is started on port which differ from 8080?

Comment (garagatyi) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java:88]: Sorry I didn't get your question.

Comment (garagatyi) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java:88]: It is not related to Che master port. Just port of an application which is inside of dev machine

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java:88]: I see. Thank you for explanation.

**Review by dmytro-ndp — APPROVED** [review]
Minor changes. Looks good.

**Pull request merged**